### PR TITLE
Debate giant menu DEVAS 1129

### PR DESCRIPTION
**Pull request by amensouissi**

### CODE REVIEW

**Comment by vincentfretin** [comment]
@paolina92 can you maybe review this? We need a second review.

File: assembl/static2/css/views/debate.scss [hunk 1]
```diff
@@ -1,3 +1,174 @@
+.debate-link.active {
+  > .navbar-menu-item {
+    background-color: $first-color;
+    color: $white !important;
```
[the remark this answers is not in the document]
Comment (amensouissi) [reply assembl/static2/css/views/debate.scss]: Yes... Please see navbar.scss

File: assembl/static2/js/app/components/debate/navigation/menuItem.jsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+          active: isSelected,
+          empty: isSelected && !displayArrow
+        })}
+        onMouseOver={this.showMenu}
+        onMouseLeave={this.hideMenu}
+      >
+        <Link
+          onClick={onClick}
+          className="menu-item"
+          to={`${get('debate', { slug: slug, phase: identifier })}${get('theme', { themeId: id })}`}
```
[the remark this answers is not in the document]
Comment (amensouissi) [reply assembl/static2/js/app/components/debate/navigation/menuItem.jsx]: Done

File: assembl/static2/js/app/components/navbar/navbar.jsx [hunk 3]
```diff
@@ -57,10 +58,23 @@ const sectionURL = ({ sectionType, url }, options) => {
 
 const SectionLink = ({ section, options }) => {
   const { title, url, sectionType } = section;
-  return url || sectionType === 'CUSTOM' ? (
-    <a href={url} className="navbar-menu-item pointer" data-text={title}>
+  if (url || sectionType === 'CUSTOM') {
```
[the remark this answers is not in the document]
Comment (amensouissi) [reply assembl/static2/js/app/components/navbar/navbar.jsx]: Yes like APOLLO_NETWORK_STATUS or PHASE_STATUS in /constans.js

[the remark this answers is not in the document]
Comment (ayyazdaniaryan) [reply assembl/static2/js/app/components/navbar/navbar.jsx]: Exactly like `PHASE_STATUS` that I saw in this PR.

File: assembl/static2/js/app/components/debate/navigation/menuTable.jsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+import { SurveyTable, IdeasTable } from './tables';
+import AllIdeasQuery from '../../../graphql/AllIdeasQuery.graphql';
+import DebateThematicsQuery from '../../../graphql/DebateThematicsQuery.graphql';
+
+const queries = {
+  survey: DebateThematicsQuery,
+  default: AllIdeasQuery
+};
+
+export function prefetchMenuQuery(client: ApolloClient, variables: Object) {
```
[the remark this answers is not in the document]
Comment (amensouissi) [reply assembl/static2/js/app/components/debate/navigation/menuTable.jsx]: No precise specification for the variables. It depends on the request...

**Pull request merged**